### PR TITLE
Sara/mkt 349 q2 launch gitlab sm and GitHub

### PR DESCRIPTION
**Pull request by s-santillan**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [ ] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [x] Redirects are added if the PR changes page URLs

### CODE REVIEW

File: docs/semgrep-app/scm.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+3. Enter the name of your parent organization.
+4. Sign out, then sign in again to enable SCM management.
+5. Click **Settings** > **SCM management** > **Configure new SCM**.
+
+<div class = "bordered">
+
+![Screenshot of SCM configuration tab](../img/app-scm.png)
+
+</div>
```
Comment (adamkvitek) [inline docs/semgrep-app/scm.md:43]: @s-santillan just not sure here about the borders. I generally think that a screenshot or anything in a numbered list should be intended with. I tried to test the difference (the screenshot above is intended and without border div).
Screenshot 2022-05-17 at 13 16 37
Screenshot 2022-05-17 at 13 16 23
:

File: docs/semgrep-app/scm.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+6. Select your SCM provider.
+7. For **GitHub Enterprise**, follow the steps below:
```
Comment (adamkvitek) [inline docs/semgrep-app/scm.md:46]:
```suggestion
7. For **GitHub Enterprise**, follow these steps:

```

File: docs/semgrep-app/scm.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    1. Create a personal access token (PAT) by following the steps outlined in this [guide to creating a PAT](https://docs.github.com/en/enterprise-server@3.1/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token).
+    2. Enter the personal access token generated into the Access Token field.
+    3. Enter your GHE base URL into the SCM Base URL.
+8. For **GitLab Self Managed**, follow the steps below:
```
Comment (adamkvitek) [inline docs/semgrep-app/scm.md:50]:
```suggestion
8. For **GitLab Self Managed**, follow these steps:
```

File: docs/semgrep-app/scm.md (hunk 2 above)
Comment (adamkvitek) [reply docs/semgrep-app/scm.md:46]: This should be a lettered list - you are choosing one of the providers.

File: docs/semgrep-app/scm.md (hunk 3 above)
Comment (adamkvitek) [reply docs/semgrep-app/scm.md:50]: This should be a lettered list - you are choosing one of the providers.

File: docs/semgrep-app/scm.md (hunk 1 above)
Comment (s-santillan) [reply docs/semgrep-app/scm.md:43]: The problem is when the theme is white, it's hard to tell when the img starts and ends. Compare the two: 


.

File: docs/semgrep-app/scm.md (hunk 2 above)
Comment (s-santillan) [reply docs/semgrep-app/scm.md]: Added it, but the indents are not the greatest (default look at the moment of docusaurus).

File: docs/semgrep-app/scm.md (hunk 1 above)
Comment (adamkvitek) [reply docs/semgrep-app/scm.md:43]: I see, let us keep borders then! @s-santillan

Comment (adamkvitek) [reply docs/semgrep-app/scm.md:43]: And let's file a Docusaurus bug later... 

For the sake of future bug report:

The following code is displayed correctly in VS Code preview:

```md
6. Select your SCM provider and follow its procedure below:
<ol type="a">
    <li>For <b>GitHub Enterprise</b>, follow the steps below:</li>
        <ol>
        <li>Create a personal access token (PAT) by following the steps outlined in this  <a href="https://docs.github.com/en/enterprise-server@3.1/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token" target="_blank">guide to creating a PAT</a>.</li>
        <li>Enter the personal access token generated into the Access Token field.</li>
        <li>Enter your GHE base URL into the SCM Base URL.</li>
    </ol>
<li>For <b>GitLab Self Managed</b>, follow the steps below:</li>
    <ol>
        <li>Create a personal access token (PAT) by following the steps outlined in this <a href="https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html" target="_blank">guide to creating a PAT</a>.</li>
        <li>Enter the personal access token generated into the Access Token field.</li>
        <li>Enter your GLSM base URL into the <b>SCM Base URL</b> field.</li>
    </ol>
</ol>

7. Ensure that your SCM integration successfully detects repositories by following the steps below:
    1. Click **Projects > Setup New Project**.
    2. Select your CI provider.
    3. Semgrep App detects repositories from your SCM integration.
```

Screenshot 2022-05-17 at 14 56 22

But the result when Docusaurus built is the following:
Screenshot 2022-05-17 at 14 57 27

**Review by adamkvitek — APPROVED** [review]

File: docs/semgrep-app/scm.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+1. Sign in to [Semgrep App](https://semgrep.dev/login).
+2. Click **[Settings](https://semgrep.dev/orgs/-/settings)** > **Create new parent organization**.
+
+<div class = "bordered">
+
+![Screenshot of settings for parent organization](../img/app-parent-org.png "Screenshot of settings for parent organization")
+
+</div>
+
+3. Enter the name of your parent organization.
```
Comment (dlukeomalley) [inline docs/semgrep-app/scm.md:35]: Do we explain what a parent organization is and/or should we provide context/guideline on what the name should be here?

In the UI it now defaults to the name of their organization. Perhaps we could instead say something like:

> Confirm the name of your organization. If needed or preferred, you can enter a new name at this time.

File: docs/semgrep-app/scm.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+<div class = "bordered">
+
+![Screenshot of SCM configuration tab](../img/app-scm.png)
+
+</div>
+
+6. Select your SCM provider.
+7. For **GitHub Enterprise**, follow these steps:
+
+    1. Create a personal access token (PAT) by following the steps outlined in this [guide to creating a PAT](https://docs.github.com/en/enterprise-server@3.1/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token).
```
Comment (dlukeomalley) [inline docs/semgrep-app/scm.md:48]: At this stage, does the user need to know what permissions they need to enable? I recall users asking this during onboarding.

File: docs/semgrep-app/scm.md (hunk 4 above)
Comment (zyannes) [reply docs/semgrep-app/scm.md:35]: It actually doesn't let the user change the organization name. It displays a dialog with the deployment name as the new parent organization name.

File: docs/semgrep-app/scm.md (hunk 5 above)
Comment (zyannes) [reply docs/semgrep-app/scm.md:48]: 👍

Comment (zyannes) [reply docs/semgrep-app/scm.md:48]: Having an explanation for the required scopes would be nice too

File: docs/semgrep-app/scm.md (hunk 4 above)
Comment (dlukeomalley) [reply docs/semgrep-app/scm.md:35]: As a V1, I could see this just being a confirmation dialog that you confirm and that then logs you out :)

Comment (s-santillan) [reply docs/semgrep-app/scm.md:35]: Okay, I'll update the text :)

File: docs/semgrep-app/scm.md (hunk 5 above)
Comment (s-santillan) [reply docs/semgrep-app/scm.md:48]: I believe this is covered in the prerequisites section above. @dlukeomalley 

For the scopes @zyannes I'll PM you to double check :)

File: docs/semgrep-app/scm.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+import MoreHelp from "/src/components/MoreHelp"
+
+# Integrating Semgrep into source code management (SCM) tools
+
+Semgrep App's Team tier supports repositories hosted on both **GitHub Enterprise (GHE)** and **GitLab Self Managed (GLSM)** plans. Create multiple integrations in different self-hosted or custom servers.
+
+## Prerequisites
+
+You need the following permissions to integrate Semgrep into GHE or GLSM servers:
```
Comment (s-santillan) [inline docs/semgrep-app/scm.md:17]: @dlukeomalley Here's the section that addresses the permissions note.

**Pull request merged**